core/runtime/bindings.rs

```diff
@@ -389,16 +388,12 @@ fn import_meta_resolve(
   let loader = module_map_rc.loader.clone();
   let specifier_str = specifier.to_rust_string_lossy(scope);
 
-  if specifier_str.starts_with("npm:") {
-    throw_type_error(scope, "\"npm:\" specifiers are currently not supported in import.meta.resolve()");
-    return;
-  }
+  let import_meta_resolve_result = {
+    let loader = loader.borrow();
+    (module_map_rc.import_meta_resolve_cb)(&loader, specifier_str, referrer)
+  };
```

Comment:
This will allow the callback to take a `&dyn ModuleLoader`

```suggestion
  let import_meta_resolve_result = {
    let loader = loader.borrow();
    let loader: &dyn ModuleLoader = loader.as_ref();
    (module_map_rc.import_meta_resolve_cb)(&*loader, specifier_str, referrer)
  };
```

Reply:
(you can probably remove the explicit type)

Reply:
Sweet, thanks! It was obviously way to late for me to figure it out 😓